src/app/admin/collectionCentresTable/CollectionCentresTable.tsx

```diff
@@ -66,16 +77,57 @@ const CollectionCentresTables: React.FC<Props> = (props) => {
         setCollectionCentreToDelete(undefined);
     };
 
+    const filters: Filter<CollectionCentresTableRow, string>[] = [
+        buildTextFilter({
+            key: "name",
+            label: "Name",
+            headers: collectionCentresTableHeaderKeysAndLabels,
+            methodConfig: { paginationType: PaginationType.Client, method: filterRowByText },
+        }),
+        buildTextFilter({
+            key: "acronym",
+            label: "Acronym",
+            headers: collectionCentresTableHeaderKeysAndLabels,
+            methodConfig: { paginationType: PaginationType.Client, method: filterRowByText },
+        }),
+    ];
+    const [primaryFilters, setPrimaryFilters] =
+        useState<Filter<CollectionCentresTableRow, string>[]>(filters);
```

Comment:
Can we 
- move `filters` outside the component
- move the `primaryFilters` state to be next to all other states at the top
?